src/components/PageLayout/Authentication/Authentication.tsx

```diff
@@ -12,24 +12,10 @@ export const Authentication: FC = () => {
   const [displayAuthenticationOverlay, setDisplayAuthenticationOverlay] = useState(false)
   const {logout, isAuthed} = AuthContainer.useContainer()
 
-  const toggleDisplayAuthenticationOverlay = () => {
-    setDisplayAuthenticationOverlay((prevDisplay) => {
-      if (!prevDisplay) {
-        // Ak sa má zobraziť overlay s login formom, input s emailom sa automaticky focusne.
-        const email = document.getElementById('login-email') as HTMLInputElement
-        email.focus()
-      }
-      return !prevDisplay
-    })
+  const toggleDisplayLoginOverlay = () => {
+    setDisplayAuthenticationOverlay(!displayAuthenticationOverlay)
```

Comment:
je to sice trochu zvlastne, ale pokial viem, best practice pre toggle statu je naozaj format funkcie s minulym stavom:
```js
    setDisplayAuthenticationOverlay((prev) => !prev)
```
to garantuje, ze je tam vzdy najviac recent hodnota